src/ConnectorRuntime.ts

```diff
@@ -98,7 +88,7 @@ export class ConnectorRuntime extends Runtime<ConnectorRuntimeConfig> {
 
     protected async createDatabaseConnection(): Promise<IDatabaseConnection> {
         if (!this.runtimeConfig.database.connectionString) {
-            this.logger.error(RuntimeErrors.startup.noDatabaseDefined());
+            this.logger.error("No database connection string provided.");
```

Comment:
Ich glaube, wir hatten hier einen Error Code geloggt, damit wir in der Dokumentation mehr Details zu dem Fehler schreiben können.

Reply:
Ich glaube in der Beschreibung von dem was in der Runtime stand waren schlechtere Infos als jetzt hier. Aber ja, vielleicht sollte es dafür nen error code, oder einfach hier inline nen link auf die Doku geben?

Reply:
Jo, gerne auch einen Link, der auf die Error Code Doku verweist. Bekommt man ja als Ergebnis des HTTP-Calls auch (denk dran, die Link-Generierungsklasse zu verwenden)